src/coord/src/coord.rs

```diff
@@ -4401,33 +4401,15 @@ impl Coordinator {
         Ok(result)
     }
 
-    async fn send_builtin_table_updates_at_offset(&mut self, updates: Vec<TimestampedUpdate>) {
```

Comment:
The thing that used this has apparently gone away, great! Looks like you can also remove the `TimestampedUpdate` struct now too.

Reply:
Great! I removed the struct and hit auto-merge